CHANGELOG.md

```diff
@@ -12,6 +12,13 @@ Every entry has a category for which we use the following visual abbreviations:
 
 ## Unreleased
 
+- ⚡️ The GitHub CI now runs on Debian Buster and produces Debian artifacts
+  instead of Ubuntu artifacts. Similarly, the Docker images we provide on
+  [dockerhub](https://hub.docker.com/r/tenzir/vast) use Debian Buster as
+  base-image. To build Docker images locally, users now must set
```

Comment:
```suggestion
  base image. To build Docker images locally, users now must set
```

Just a low-prio nit. If this is technical Docker term, you also do `base-image`.

Reply:
The official Docker docs spell it without dash: https://docs.docker.com/develop/develop-images/baseimages/

Reply:
Fwiw if you don't wanna do this rn, I'll do a pass over all items for the release prep anyways.